src/js/utils.js

```diff
@@ -48,3 +48,11 @@ export function timeDifferenceFuture(current, previous) {
     return $.i18n('years', Math.round(elapsed / msPerYear))
   }
 }
+
+export const getNodesDeep = (obj, depth) => {
+  let result = obj
+  for (let i = 0; i < depth; i++) {
```

Comment:
Please remember that each block like `for` should be separated with one line.

```javascript
let result = obj

for (let i = 0; i < depth; i++) result = result.parentNode

return result
```

Is it guaranteed to be have as many as `depth` nodes?

Reply:
From what i’ve seen, the depth required to get to the node holding 1 individual tweet remain so the same no matter how the twitter widget is generated

Reply:
(and no matter the contents of the tweet)

Reply:
Alright then.